README.md

```diff
@@ -1347,31 +1347,21 @@ withConnection(
 .fork(console.error, console.log);
 ```
 
-In the case that a hooked Future is *cancelled* after the resource was acquired,
-`dispose` will be executed and immediately cancelled. This means that rejections
-which may happen during this disposal are **silently ignored**. To ensure that
-resources are disposed during cancellation, you might synchronously dispose
-resources in the `cancel` function of the disposal Future:
+When a hooked Future is cancelled while acquiring its resource, nothing else
+will happen. However, once the resource has been acquired, the `dispose`
+Future is guaranteed to run.
 
-<!-- eslint-disable no-unused-vars -->
-```js
-var closeConnection = conn => Future((rej, res) => {
-
-  //We try to dispose gracefully.
-  conn.flushGracefully(err => {
-    if(err === null){
-      conn.close();
-      res();
-    }else{
-      rej(err);
-    }
-  });
+In the case where the consumption Future settles, the disposal is forked and
+awaited. If the disposal Future rejects or encounters an exception, it will
+replace the state of the consumption Future. Otherwise the rejection reason or
+resolution value of the consumption Future will determine the final outcome.
```

Comment:
Is “consumption” being used as an adjective here? This doesn't sound right to me.

Reply:
I need to quick way to refer to "the Future returned by the `consume` function".

Reply:
Fair enough.